keyboards/geminate60/geminate60.h

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2020 Weirdo
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_ortho_5x15( \

```

Comment:
```suggestion
#define LAYOUT( \
```

Also, since not all of the keys are being used here, you'd want to add `KC_NO` to the bottom half, here.

Reply:
> Also, since not all of the keys are being used here, you'd want to add `KC_NO` to the bottom half, here.

I've added KC_ NO, thank you.